Pandas_and_visualization/pandas_and_visualization_task.py

```diff
@@ -0,0 +1,118 @@
+import pandas as pd
+import matplotlib.pyplot as plt
+import re
+import seaborn as sns
+
+""" Напишите функции read_gff и read_bed6 для чтения соответствующих форматов. 
+    Они должны возвращать датафреймы как в примере (картинка Example1), но имена колонок можно сделать любыми.
+"""
+pd.set_option("display.max_columns", None)
+
+
+def read_gff(path_to_gff_file):
+    columns = ["chromosome", "source", "type", "start", "end", "score", "strand", "phase", "attributes"]
+    gff_df = pd.read_csv(path_to_gff_file, sep="\t", names=columns, skiprows=1)
```

Comment:
Твой код учитывает только случай, где комментарий занимает только первую строку. Однако чаще всего в файлах данного формата могут быть комментарии на много строк, их нужно учитывать. Благо в пандасе это делается супер просто
```suggestion
    gff_df = pd.read_csv(path_to_gff_file, sep="\t", names=columns, comment="#")
```